### DISCUSSION TITLE
Welcome to Flareact Discussions!

### DISCUSSION THREAD

**Discussion by jplhomer** [flareact/flareact#72] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by idl3** [comment]
Hey everyone, I'm Ernest from Singapore, I lead a team at Grain and also in the midst of working on a new SaaS software. Fascinated about Cloudflare Workers and also very interested to see what I can do with Flareact.

I was initially looking at ideas to develop a multi-tenanted front-end solution and stumbled on Flareact. The basic premise was to load a performant React Application but inject custom configurations/css (styling, layout) and also data on the fly.

**Reply by jplhomer** [reply]
Nice to meet you, Ernest! Awesome to hear about your interest in Flareact. I hadn't considered the multitenant use case. Excited to see what you build.
